Serializer/Handler/DateHandler.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/*
+ * This file is part of the opcart software.
+ *
+ * (c) 2018, OpticsPlanet, Inc.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Mango\Bundle\JsonApiBundle\Serializer\Handler;
+
+use JMS\Serializer\Handler\DateHandler as BaseDateHandler;
+use Mango\Bundle\JsonApiBundle\MangoJsonApiBundle;
+
+/**
+ * DateHandler handler to add the same handlers for dates in json:api format as for json format
+ *
+ * @copyright 2018 OpticsPlanet, Inc
+ * @package   SabioApp
```

Comment:
this comment should be removed

Reply:
@oleg-andreyev, please comment on https://github.com/ecentria/JsonApiBundle/pull/1

Reply:
What you are referring to? Link target is PR#1

Reply:
@oleg-andreyev correct. Please leave your comment in #1